doc/installation_and_administration_guide/configuration.md

```diff
@@ -310,7 +312,59 @@ cygnusagent.channels.mkafka-channel.transactionCapacity = 100
 
 [Top](#top)
 
-##<a name="section4"></a>`log4j.properties`
+##<a name="section4"></a>`flume-env.sh`
+The file `flume-env.sh`  can be instantiated from a template given in the Cygnus repository, `conf/flume-env.sh.template`. 
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/dab65c18673004b631398d98a60c8887dae24939